mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -222,6 +223,53 @@ pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
         },
     };
 
+    if let SwapMsg::WatcherMessage(watcher_data) = msg.0 {
+        spawn(async move {
```

Comment:
If I'm not mistaken, we don't check if any Swap watcher has been spawned for the same `uuid` already.
Malicious Taker node can spam the network with these messages.